app/components/friend-code.jsx

```diff
@@ -1,16 +1,28 @@
-import { Link }    from 'react-router';
-import { connect } from 'react-redux';
+import { Link }     from 'react-router';
+import { connect }  from 'react-redux';
 
 import { ReactGA } from '../utils/analytics';
 
 export function FriendCode ({ session, user }) {
   const ownPage = session && session.id === user.id;
 
+  let editAccountBtn = null;
+
+  if (ownPage) {
+    editAccountBtn = (
+      <Link to="/account" onClick={() => ReactGA.event({ action: 'click edit friend code', category: 'User' })}><i className="fa fa-pencil" /></Link>
+    );
+  }
+
   return (
-    <h2>
-      FC: <span className={user.friend_code ? '' : 'fc-missing'}>{user.friend_code || 'XXXX-XXXX-XXXX'}</span>
-      {ownPage ? <Link to="/account" onClick={() => ReactGA.event({ action: 'click edit friend code', category: 'User' })}><i className="fa fa-pencil" /></Link> : null}
-    </h2>
+    <div>
```

Comment:
would be nice to upgrade to v16 of react at some point so we can use [fragments](https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html) 😄

Reply:
TIL

Reply:
we use them in lens!!!